tree/30_generic_methods/file_from_template_jinja2.cf

```diff
@@ -0,0 +1,45 @@
+#####################################################################################
+# Copyright 2014 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @name File from a mustache template
+# @description This is a bundle to build a file from a mustache template
+# @agent_version >=3.6
+#
+# @parameter source_template Source file containing a template to be expanded
+# @parameter destination     Destination file
+#
+# @class_prefix file_from_template
+# @class_parameter destination
+
+bundle agent file_from_template_jinja2(source_template, destination)
+{
+
+  vars:
+      "jinja2_script" string => "/home/amousset/JINJA/cli.py";
```

Comment:
Hmmm. Since we added some python external dependencies in modules/ the other day, why not use that path? They will already be copied... But... wait... why do we need to copy them?